usage.go

```diff
@@ -0,0 +1,83 @@
+package lago
+
+import (
+	"context"
+	"encoding/json"
+)
+
+type UsageRequest struct {
+	client *Client
+}
+
+type UsageCustomerType string
+
+const (
+	UsageCompanyCustomerType    UsageCustomerType = "company"
+	UsageIndividualCustomerType UsageCustomerType = "individual"
+)
+
+type UsageListInput struct {
+	Currency					string				`json:"currency,omitempty"`
+	CustomerCountry     		string 				`json:"customer_country,omitempty,string"`
+	CustomerType     			UsageCustomerType	`json:"customer_type,omitempty,string"`
+	FromDate            		string          	`json:"from_date,omitempty"`
+	ToDate              		string          	`json:"to_date,omitempty"`
+	IsBillableMetricRecurring 	bool        	    `json:"is_billable_metric_recurring,omitempty"`
+	TimeGranularity    			string   	    	`json:"time_granularity,omitempty"`
+	ExternalCustomerID 			string      	    `json:"external_customer_id,omitempty"`
+	ExternalSubscriptionID		string				`json:"external_subscription_id,omitempty"`
+	BillableMetricCode			string				`json:"billable_metric_code,omitempty"`
+	PlanCode					string				`json:"plan_code,omitempty"`
+}
+
+type UsageResult struct {
+	Usage  *Usage  `json:"usage,omitempty"`
```

Comment:
I believe that's not part of the OpenAPI doc: https://github.com/getlago/lago-openapi/pull/368/files#diff-49fb99fdd973c9f517cdc7e3b701a5df89e5c9a2f260603bb299a0c759021d69R5